client/src/app/pages/archetypes/archetypes-page.tsx

```diff
@@ -68,6 +68,7 @@ const Archetypes: React.FC = () => {
   const [archetypeToEdit, setArchetypeToEdit] = useState<Archetype | null>(
     null
   );
+  const [reviewToEdit, setReviewToEdit] = React.useState<number | null>(null);
```

Comment:
`reviewToEdit` doesn't seem to be used anywhere.  Would it be enough to just do the history push?